### DISCUSSION TITLE
Fedora package

### DISCUSSION THREAD

**Discussion by 0xMRTT** [mjakeman/extension-manager#245] [Show and tell]
I've created a rpm package for extension-manager

https://github.com/0xMRTT/extension-manager-rpm/blob/main/extension-manager/extension-manager.spec

https://copr.fedorainfracloud.org/coprs/0xmrtt/extension-manager/

**Comment by mjakeman** [comment]
Great work, thanks a lot!

**Reply by 0xMRTT** [reply]
You can add it in the third party packages section. A review request is in process for pushing in the fedora repository.
